### ISSUE TITLE
Oversized Smoke Particles

### ISSUE THREAD

**Issue by Ellasent** [shaddatic/sa2b-render-fix#75]
Even though the particle size correction fixes most of the particle sizes, it seems that the smoke in Prison Lane and presumably Iron Gate were manually corrected for Battle, causing them to be way too big when the particle size is doubled with Render Fix.
Makes sense they'd do that given how many more smoke objects are present in the Battle versions of these stages, probably making the undersized smoke particle more noticeable during development.
DC
![image](https://github.com/shaddatic/sa2b-render-fix/assets/62478552/f603f433-67af-4d7c-a6d1-741b0c4a3d1c)
GCN
![image](https://github.com/shaddatic/sa2b-render-fix/assets/62478552/76d9537f-f788-486e-b923-c0f6ceeedf05)
Render Fix Particle Size Correction
![image](https://github.com/shaddatic/sa2b-render-fix/assets/62478552/51d4e1e9-3e29-4222-b266-6ed18ced069a)

**Comment by shaddatic** [comment]
Yeah, I noticed this before but had no easy way of fixing it and I was averse to changing too many game files due to potential mod conflicts. Now that Render Fix throws a warning if it's not first though, I will give this another look
